src/components/RadioButton/RadioButton.scss

```diff
@@ -60,11 +60,15 @@
   &__radio-label {
     line-height: 21px;
     font-weight: 400;
+    font-size: var(--font-size-general-label);
     @include theme-prop(color, primary-text-color);
   }
 
   &.disabled {
     cursor: not-allowed;
+    .monday-style-radio-button-component__radio-label {
+      @include theme-prop(color, disabled-text-color);
```

Comment:
we don't use theme-prop for new css anymore, you can just use the css variable like in other places.